shells/chrome/src/main.js

```diff
@@ -21,9 +21,6 @@ function createPanelIfReactLoaded() {
   chrome.devtools.inspectedWindow.eval(`!!(
     Object.keys(window.__REACT_DEVTOOLS_GLOBAL_HOOK__._renderers).length || window.React
   )`, function(pageHasReact, err) {
-    if (!pageHasReact || panelCreated) {
```

Comment:
Why do you return early here? With this change in place, it looks like the devtools will load the React tab on pages where React isn't present. This could be undesirable.